src/test/resources/data/StudentFeedbackSubmissionEditPageActionTest.json

```diff
@@ -1456,4 +1456,4 @@
         }
         
     }
-}
\ No newline at end of file
+}
```

Comment:
Quite a number of these additional newlines at the end of the file, but I don't think it really hurts the PR so it should be okay.